File: prog-examples/09-go/printheadlines.go
// Datei: prog-examples/09-go/printheadlines.go
package main
import (
	"fmt"
	"github.com/mmcdole/gofeed"
)

func main() {
	fp := gofeed.NewParser()
	feed, err := fp.ParseURL("https://www.heise.de/newsticker/heise-atom.xml")

	if err != nil {
		panic(err)
	}

	for _, item := range feed.Items {
		fmt.Printf("* [%s]: %s\n",
	      item.PublishedParsed.Local().Format("2006-01-02 15:04:05"),
		  item.Title)
	}
}

File: prog-examples/06-python/printheadlines.py
#!/usr/bin/env python3
# Datei: prog-examples/06-python/printheadlines.py
import feedparser
import time

feed = feedparser.parse("https://www.heise.de/newsticker/heise-atom.xml")

[ print("* [%s]: %s" %
    (time.strftime("%Y-%m-%d %H:%M:%S", entry.published_parsed),
     entry.title))
    for entry in feed.entries ]
